contracts/utils/SafeAccessControlEnumerable.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+
+import "./PendingOwnable.sol";
+
+error SafePausableAccessControlEnumerable__SenderMissingRoleAndIsNotOwner(
+    bytes32 role,
+    address sender
+);
+
+abstract contract SafeAccessControlEnumerable is
```

Comment:
is the name of the contract `SafePausableAccessControlEnumerable` or `SafeAccessControlEnumerable`? asking b/c the name in the error is different from the contract name